data/phi0959/phi001/phi0959.phi001.perseus-eng2.xml

```diff
@@ -10,6 +10,7 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
 				<principal>Gregory Crane</principal>
 				<respStmt>
 					<resp>Prepared under the supervision of</resp>
+					<name>Anne Mahoney</name>
```

Comment:
@AlisonBabeu 
This is fine here since we know the work done, but this is generally alphabetical.